app/models/manageiq/providers/redhat/inventory/persister/definitions/infra_group/hosts_collections.rb

```diff
@@ -10,58 +10,17 @@ def add_hosts
       )
 
       if targeted?
-        builder.add_properties(:arel => manager.hosts.where(:ems_ref => manager_refs)) if manager_refs.present?
+        builder.add_targeted_arel(
+          lambda do |_inventory_collection|
+            manager.hosts.where(:ems_ref => references(:hosts)) # TODO why are we using ems_ref instead of uid_ems?
```

Comment:
@borod108  @agrare what is the reason of using :ems_ref here? The manager_ref for :hosts is uid_ems, should we use :ems_ref instead?

Reply:
Hm looks like this is hard-coded to use `:ems_ref` as the manager_ref [link](https://github.com/ManageIQ/manageiq-providers-ovirt/blob/master/app/models/manageiq/providers/redhat/inventory/collector/target_collection.rb#L133-L135), should probably be switched to allow `:uid_ems` as well.  @borod108 would this always return `[]` if the manager_ref isn't `:ems_ref`?

Reply:
Hm okay I take it back, looks like it is using the ems_ref:
```
(byebug) target.try(:manager_refs_by_association).try(:[], collection)
{:ems_ref=>#<Set: {"/ovirt-engine/api/hosts/11089411-53a2-4337-8613-7c1d411e8ae8"}>}
```